docs/source/articles/add-catalog-customizations.html.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Add Catalog Customizations
+excerpt: This guide shows you how to implement user-defined customizations for the products in your catalog.
+---
+
+# Add Catalog Customizations
+
+Customizations are pre-defined fields for your product that customers can use
+to create personalized, custom versions of that product. Some examples of how
+these have been used include:
+
+* Custom engravings for jewelry
+* Monograms for shirts
+* Email addresses for digital gift cards 
+
+## Step 1: Create a `Catalog::Customizations` subclass
+
+The first step is to subclass `Workarea::Catalog::Customizations` with
+something descriptive of your customization. In this case, we'll be using the
+concept of an "engraving" for jewelry. Create a file named
+**app/models/workarea/catalog/customizations/engraving.rb** and populate it
+with the following class definition:
+
+```ruby
+module Workarea
+  module Catalog
+    class Customizations::Engraving < Customizations
+      customized_fields :initials
```

Comment:
Can you add some inline explanation of what you'd want to put here and why?

Reply:
oh sorry i thought that was grouped with your other change, i'll commit it.